experimental/overhead_matching/grounding_sam_demo.ipynb

```diff
@@ -0,0 +1,176 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib ipympl\n",
+    "import os\n",
+    "\n",
+    "import common.torch as torch\n",
+    "import requests\n",
+    "import cv2\n",
+    "import supervision as sv\n",
+    "from supervision.draw.color import ColorPalette\n",
+    "import numpy as np\n",
+    "from PIL import Image\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "import experimental.overhead_matching.grounding_sam as gs\n",
+    "import importlib\n",
+    "importlib.reload(gs)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = gs.GroundingSam()\n",
+    "\n",
+    "image_url = \"https://s3.us-east-1.amazonaws.com/images.cocodataset.org/train2017/000000146439.jpg\"\n",
+    "image = Image.open(requests.get(image_url, stream=True).raw)\n",
+    "img = np.array(image.convert('RGB'))\n",
+    "\n",
+    "plt.figure()\n",
+    "plt.imshow(img)\n",
+    "\n",
+    "# VERY important: text queries need to be lowercased + end with a dot\n",
+    "queries = ['cat', 'keyboard', 'piano', 'speaker']\n",
+    "\n",
+    "results = model.detect_queries(image = img, queries = queries)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(results['dino_results'])\n",
+    "\n",
+    "results['dino_results']['boxes'][0].tolist()\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
```

Comment:
are you manually clearing your notebooks before you commit them?

Reply:
I'm using a tool called `nbstripout` https://github.com/kynan/nbstripout